app/core/DeeplinkManager/CoreLinkNormalizer.ts

```
@@ -85,8 +85,9 @@ export class CoreLinkNormalizer {
         isPrivateLink: false, // Will be determined by signature verification
         requiresAuth: isAuthRequiredAction(action),
       };
-    } catch (error) {
-      // Return invalid link representation
+    } catch (_error) {
+      // for some reason SonarQube doesn't think this
+      // counts as "handling" the error
```

Because it's actually what happens. You do things but the error variable is not used. So for Sonar, you are doing something but ignoring the error. Which is true, you kinda ignore it because you don't even look at waht error it is and just fallback. So you can mark this sonar issue as accepted.